src/traces/sankey/plot.js

```diff
@@ -62,9 +64,38 @@ function nodeNonHoveredStyle(sankeyNode, d, sankey) {
 }
 
 function linkHoveredStyle(d, sankey, visitNodes, sankeyLink) {
-    sankeyLink.style('fill-opacity', function(l) {
+    if(!linkStyleInitialized) {
+        // Figure out whether the user has provided their own sankey-link-hover style.
+        var styleExists = false;
+        for(var i = 0; i < document.styleSheets.length; i++) {
+            var rules = document.styleSheets[i].cssRules;
```

Comment:
Here I get the following error in the console when start using hover:
```
Uncaught DOMException: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
```

Reply:
Hi @archmoj , thanks for having a look at this. I guess this is due to cross-origin resource sharing policy preventing your browser from looking at css sheets from different domains. I didn't encounter it personally, but I've added a try/catch block to allow the code to skip over css sheets that it's not allowed to read. Does it work now on your end?